web/app/js/components/util/Utils.js

```diff
@@ -74,3 +75,11 @@ export const toClassName = name => {
   if (!name) return "";
   return _.lowerCase(name).replace(/[^a-zA-Z0-9]/g, "_");
 };
+
+/*
+* Instructions for adding deployments to service mesh
```

Comment:
Thanks for adding! I think that instead of utils, we should store specific copy like this in a file dedicated to that. but I think we should leave for a follow up branch

Reply:
sounds good, will leave it for now :)